ios/Samples/WesterosEmployees-hybrid/App/WesterosEmployees-hybrid/Resources/css/blog.css

```diff
@@ -0,0 +1,59 @@
+html, body {
+    max-width: 100%;
+    overflow-x: hidden;
+}
+
+body{
+    background: #E44740;
+    color: #FFF;
+    margin: 0px;
+    font-size: 18px;
+}
+
+#wrapper {
+    padding-top: 0em;
+}
+
+.col-md-12 {
+    padding: 0px;
+}
+
+.portlet-content-editable{
+    padding: 0px;
+}
+
+.portlet-decorate .portlet-content{
+    border: 0px;
+    margin-top: -50px;
+    background: #E44740;
+}
+
+a {
+    color: #FFF !important;
+}
+
+p {
+    margin: 0px 20px;
+}
+
+
+.author-content{
+    margin-left: 20px;
+}
+
+.user-icon{
```

Comment:
pass a formatter like webstorm or idea to have more consistent organization of the css files